project/Dependencies.scala

```diff
@@ -32,8 +32,8 @@ object Dependencies {
 
   object PlayVersion {
     val V27 = PlayVersion("27", "com.typesafe.play", "2.7.9")
-    val V28 = PlayVersion("28", "com.typesafe.play", "2.8.20")
-    val V29 = PlayVersion("29", "com.typesafe.play", "2.9.0", supportsScala3 = true)
+    val V28 = PlayVersion("28", "com.typesafe.play", "2.8.21")
+    val V29 = PlayVersion("29", "com.typesafe.play", "3.0.0", supportsScala3 = true)
```

Comment:
The upgrade for `V29` from 2.9.0 to 3.0.0 is wrong here - we don't want it to jump major versions.

See https://github.com/guardian/play-googleauth/pull/206